VKClone/ViewControllers/NewsViewController.swift

```diff
@@ -0,0 +1,232 @@
+                    Photo(photoName: "photo5", likesCount: 19)
+                ]
+            ), Friend(
+                friendNickName: "DFriend5",
+                friendImageName: Constants.friendImageNameText,
+                friendGroupName: "Команда ВКонтакте",
+                photos: [
+                    Photo(photoName: "photo6", likesCount: 10),
+                    Photo(photoName: "photo7", likesCount: 10),
+                    Photo(photoName: "photo1", likesCount: 10)
+                ]
+            ), Friend(
+                friendNickName: "DoAriend6",
+                friendImageName: Constants.friendImageNameText,
+                friendGroupName: "",
+                photos: [
+                    Photo(photoName: "photo2", likesCount: 10),
+                    Photo(photoName: "photo3", likesCount: 10),
+                    Photo(photoName: "photo4", likesCount: 10)
+                ]
+            )
+        ]
+    }
+}
+
+/// UITableViewDelegate, UITableViewDataSource
+extension NewsViewController {
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        news.count
+    }
+
+    override func tableView(
+        _ tableView: UITableView,
+        cellForRowAt indexPath: IndexPath
+    ) -> UITableViewCell {
+        guard let cell = tableView.dequeueReusableCell(
+            withIdentifier: NewsViewCell.identifier,
+            for: indexPath
+        ) as? NewsViewCell else {
+            return UITableViewCell()
+        }
+        cell.configure(with: news[indexPath.item])
+        return cell
+    }
+
+    override func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        UITableView.automaticDimension
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        tableView.reloadData()
+    }
+
+    override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        performSegue(withIdentifier: "goToPhotoGallerySegue", sender: self)
+    }
+}
+
+/// константы
+extension NewsViewController {
+    enum Constants {
+        static let newsTextOne = "asdasd ad asd ada dad asd asd asd asd adas" +
+            "dasd asd asd asd asd asda sda sda sdas dasd asd asd asd "
+        static let newsTextTwo = "text two text two text two"
+        static let newsTextThree = "text three text three text three text three" +
+            "dasd asd asd asd asd asda text three text three" +
+            "dasd asd asd asd asd asda text three text three"
+        static let newsTextFour = "newsTextFour newsTextFour newsTextFour" +
+            "newsTextFour newsTextFour newsTextFour newsTextFournewsTextFour" +
+            "newsTextFour newsTextFour newsTextFour newsTextFournewsTextFour"
+        static let newsTextFive = "newsTextFive asd asd adas" +
+            "dasd newsTextFive newsTextFive newsTextFive newsTextFive newsTextFive "
```

Comment:
```suggestion
        static let newsTextOne = "asdasd ad asd ada dad asd asd asd asd adas" +
            "dasd asd asd asd asd asda sda sda sdas dasd asd asd asd "
        static let newsTextTwo = "text two text two text two"
        static let newsTextThree = "text three text three text three text three" +
            "dasd asd asd asd asd asda text three text three" +
            "dasd asd asd asd asd asda text three text three"
        static let newsTextFour = "newsTextFour newsTextFour newsTextFour" +
            "newsTextFour newsTextFour newsTextFour newsTextFournewsTextFour" +
            "newsTextFour newsTextFour newsTextFour newsTextFournewsTextFour"
        static let newsTextFive = "newsTextFive asd asd adas" +
            "dasd newsTextFive newsTextFive newsTextFive newsTextFive newsTextFive "
```
сделать через многострочные комментарии, пример Ани скидывала в чат сегодня